lib/src/revset.rs

```diff
@@ -377,8 +377,8 @@ pub enum RevsetExpression {
     Roots(Rc<RevsetExpression>),
     VisibleHeads,
     PublicHeads,
-    Branches,
-    RemoteBranches,
+    Branches(String),
+    RemoteBranches(String, String),
```

Comment:
Might be better to use struct form: `RemoteBranches { branch_needle: String, remote_needle: String }`.

Reply:
Neat, done!